packages/pharaoh/lib/src/http/response.dart

```diff
@@ -182,12 +183,13 @@ class Response extends Message<shelf.Body?> implements $Response {
     );
   }
 
+  /// [data] should be json-encodable
   @override
   Response json(Object? data) {
     late Object result;
     try {
       if (data is Set) data = data.toList();
-      result = jsonEncode(data);
+      result = jsonEncode(data, toEncodable: customEncoder);
```

Comment:
Let's remove this. Let's not do a custom encode anymore.

Also the comment should be on the interface. Not inside the implementation class.